charts/openshift-metering/templates/hadoop/hadoop-scripts.yaml

```diff
@@ -3,6 +3,14 @@ kind: ConfigMap
 metadata:
   name: hadoop-scripts
 data:
+  azure_secret_copy.sh: |
+    #!/bin/bash
+    set -x
+    cp -v -L -r -f /hadoop-starting-config/* /hadoop-config/
+    export AZURE_PROPERTY_NAME="fs.azure.account.key."$AZURE_STORAGE_ACCOUNT_NAME".blob.core.windows.net"
```

Comment:
Ok, so I think that you can use the following `faq` command instead of setting an extra env-var. To concatenate the strings you need to have parenthesis around it:

```
faq '.configuration.property+=[{name: ("fs.azure.account.key." + env.AZURE_STORAGE_ACCOUNT_NAME + ".blob.core.windows.net"), value: env.AZURE_SECRET_ACCESS_KEY }]' core-site.xml
```